.ci/deps.sh

```diff
@@ -20,6 +20,7 @@ go get -u sourcegraph.com/sqs/goreturns
 go get -u golang.org/x/tools/cmd/gotype
 go get -u github.com/kisielk/errcheck
 go get -u github.com/BurntSushi/toml/cmd/tomlv
+go get -u github.com/alecthomas/gometalinter
```

Comment:
It is better to install stable version, see https://github.com/alecthomas/gometalinter#installing